package-lock.json

```diff
@@ -108,6 +108,7 @@
       }
     },
     "../ocean.js": {
```

Comment:
the package-lock has references to symlinked ocean.js which might make things go weird

Reply:
this is irrelevant for the moment, we each work with local ocean.js. It will be sorted up once we have a release of lib